Cytoo/peaksToColonies.m

```diff
@@ -36,10 +38,31 @@
 end
 pts=alldat(:,1:2);
 
-[~, S]=alphavol(pts,100);%original value 100
-groups=getUniqueBounds(S.bnd);   % S.bnd - Boundary facets (Px2 or Px3)
-allinds=assignCellsToColonies(pts,groups);
-alldat=[alldat full(allinds)];
+% [~, S]=alphavol(pts,100);%original value 100            
+% groups=getUniqueBounds(S.bnd);   % S.bnd - Boundary facets (Px2 or Px3)
+% 
+% 
+% allinds=assignCellsToColonies(pts,groups);
+% alldat=[alldat full(allinds)];
+
+%------------------------------------------ this is where the chice is made
+if  exist('coltype','var') && coltype == 1    %analysis for the single cell data                           
+    disp('Running the SC colony analysis');
+    allinds=NewColoniesAW(pts);
+    alldat = [alldat, allinds];
+    groups = max(allinds);
+    
+else if  coltype == 0 || ~ exist('coltype','var') % analysis for the circular colonies data; defaule it no coltype variable is specified in the parameterfile
```

Comment:
This statement will cause a crash if coltype does not exist. You need to check for existence first, then check the value.